### PR TITLE
Admin dashboard for basic stats overiew

### PR DESCRIPTION
**Pull request by ushkarev**

### CODE REVIEW

File: mtp_api/apps/core/views.py [hunk 1]
```diff
@@ -6,39 +6,86 @@
...
+class AdminViewMixin:
     """
-    Django admin view which calls load_test_data management command
+    Mixin for custom MTP django admin views
     """
-    form_class = RecreateTestDataForm
-    template_name = 'core/recreate_test_data.html'
-    success_url = reverse_lazy('mtp-admin:recreate_test_data')
+    disable_in_production = False
+    superuser_required = False
```
Comment (ibrechin) [inline mtp_api/apps/core/views.py]: Should these be overridden in ```RecreateTestDataView```?

File: mtp_api/apps/core/dashboards.py [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+    class Media:
+        css = {
+            'all': ('core/css/transaction-report.css',)
+        }
+
+    def __init__(self, received_at=None):
+        if not received_at:
+            self.title = _('Today’s transaction report')
+            received_at = now().date()
```
Comment (ibrechin) [inline mtp_api/apps/core/dashboards.py]: Should the default rather be yesterday, as this is when the most recent transactions will be dated?

File: mtp_api/apps/mtp_auth/views.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from django.db.transaction import atomic
 from django.forms import ValidationError
 from django.http import Http404
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```
Comment (ibrechin) [inline mtp_api/apps/mtp_auth/views.py]: Por que?

File: mtp_api/apps/core/views.py (hunk 1 above)
Comment (ushkarev) [inline mtp_api/apps/core/views.py]: quite right

File: mtp_api/apps/core/dashboards.py (hunk 2 above)
Comment (ushkarev) [inline mtp_api/apps/core/dashboards.py]: quite right, i was being stupid

File: mtp_api/apps/mtp_auth/views.py (hunk 3 above)
Comment (ushkarev) [inline mtp_api/apps/mtp_auth/views.py]: parce que le python 3

Comment (ibrechin) [inline mtp_api/apps/mtp_auth/views.py]: bueno

**Pull request merged**